.github/workflows/delete-untagged.yml

```
@@ -15,7 +15,7 @@ jobs:
         package: ["bess_build", "upf-epc-bess", "upf-epc-pfcpiface"]
     steps:
       - name: Delete image
-        uses: bots-house/ghcr-delete-image-action@9dee3adb4e4082ef06710f3d8aa9f7e73f89d189
+        uses: bots-house/ghcr-delete-image-action@3827559c68cb4dcdf54d813ea9853be6d468d3a4
```

@badhrinathpa, @thakurajayL do you know why this GitHub Action is using a specific commit ID instead of using a version number? FYI, the latest version for `ghcr-delete-image-action` is 1.1.0. Is it because this Action only has 2 releases, so it is better to update it based on the commit IDs?
```suggestion
        uses: bots-house/ghcr-delete-image-action@v1.1.0
```